source/windows/secure_channel_tls_handler.c

```diff
@@ -1144,8 +1144,7 @@ static int s_do_application_data_decrypt(struct aws_channel_handler *handler) {
         } else {

```

Comment:
I'm noticing that many lines up, if this is false: https://github.com/awslabs/aws-c-io/blob/a3555c86bd10149a1ab0c1b2810756ec54cfbe3a/source/windows/secure_channel_tls_handler.c#L1086

then this function returns AWS_OP_ERR without having ever called `aws_raise_error()`.

Let's add an `else` branch and have it do `AWS_IO_TLS_ERROR_READ_FAILURE` there as well

Reply:
I have added a raise_error in that function.